src/pages/TaskQueue.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { Table } from "antd";
+import { ColumnProps } from "antd/es/table";
+import { useQuery } from "@apollo/react-hooks";
+import {
+  DistroTaskQueueQuery,
+  DistroTaskQueueQueryVariables,
+  TaskQueueItem,
+} from "gql/generated/types";
+import { DISTRO_TASK_QUEUE } from "gql/queries";
+import { TableContainer, PageWrapper } from "components/styles";
+import { Body } from "@leafygreen-ui/typography";
+
+export const TaskQueue = () => {
+  const { data: taskQueueItemsData, loading } = useQuery<
+    DistroTaskQueueQuery,
+    DistroTaskQueueQueryVariables
+  >(DISTRO_TASK_QUEUE, { variables: { distroId: "rhel71-power8-small" } });
```

Comment:
this is meant to be static for now. will update based on url param in future pr.

Reply:
will this be the default param in the URL?

Reply:
no